strawberry/relay/types.py

```diff
@@ -408,17 +408,14 @@ def resolve_id_attr(cls) -> str:
             base_namespace = sys.modules[base.__module__].__dict__
 
             for attr_name, attr in getattr(base, "__annotations__", {}).items():
-                try:
-                    evaled = eval_type(
-                        ForwardRef(attr) if isinstance(attr, str) else attr,
-                        globalns=base_namespace,
-                    )
-                except TypeError:
-                    # Some ClassVar might raise TypeError when being resolved
-                    # on some python versions. This is fine to skip since
-                    # we are not interested in ClassVars here
+                if is_classvar(base, attr):
```

Comment:
I'd retain the comment about this though :D

Reply:
Done! :)